tests/check-http-method.spec.js

```diff
@@ -0,0 +1,18 @@
+const HTTP_METHODS = require('../lib/http-methods');
+const checkHttpMethod = require('../lib/check-http-method');
+
+describe('checkHttpMethod function ->', () => {
+    HTTP_METHODS.forEach(method => {
+        it(`should treat HTTP method "${method}" as a correct method`, done => {
+            const result = checkHttpMethod(method);
+            expect(result).toBe(true);
+            done();
```

Comment:
I am not sure why we are using `done` callback in this case.
`checkHttpMethod` works synchronously.
I've used `done` callback in the previous unit tests, because request -> response cycle is asynchronous in it's nature.